cluster/gce/util.sh

```diff
@@ -333,7 +333,7 @@ function upload-tars() {
     # Ensure the buckets are created
     if ! gsutil ls "${staging_bucket}" >/dev/null; then
       echo "Creating ${staging_bucket}"
-      gsutil mb -l "${region}" -p "${PROJECT}" --retention "${retention_period}" "${staging_bucket}"
+      gsutil mb -l "${region}" -p "${PROJECT}" "${staging_bucket}"
```

Comment:
probably we should add `gsutil retention clear "${staging_bucket}"` below this if block?

Reply:
Preferable to do this manually to quickly unblock ?

Reply:
Only if it's actually easier to enumerate the buckets than adding this one line?

Reply:
+1 add the line Ben mentions.